addons/test_mail_sms/models/test_mail_sms_models.py

```diff
@@ -16,11 +16,10 @@ class MailTestSms(models.Model):
     subject = fields.Char()
     email_from = fields.Char()
     phone_nbr = fields.Char()
-    mobile_nbr = fields.Char()
```

Comment:
Why changing test model ? it tests models having several phone numbers ... it should still be supported.